pennylane/circuit_graph.py

```diff
@@ -346,18 +364,80 @@ def layers(self):
 
         return layers
 
-    def iterate_layers(self):
+    def iterate_parametrized_layers(self):
         """Parametrized layers of the circuit.
 
         Returns:
             Iterable[LayerData]: layers with extra metadata
         """
         # iterate through each layer
-        for ops, param_inds in self.layers:
+        for ops, param_inds in self.parametrized_layers:
             pre_queue = self.ancestors_in_order(ops)
             post_queue = self.descendants_in_order(ops)
             yield LayerData(pre_queue, ops, tuple(param_inds), post_queue)
 
+    def greedy_layers(self):
+        """Greedily collected layers of the circuit. Empty slots are filled with ``None``.
+
+        Returns:
+            Tuple[list[list[~.Operation]], list[list[~.Observable]]]:
+                Tuple of the circuits operations and the circuits observables, both indexed
```

Comment:
```suggestion
            Tuple of the circuits operations and the circuits observables, both indexed
```

Reply:
(since there is only ever one return type, continuation lines shouldn't be indented)